src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements Area {
```

Comment:
The class name 'Figure' is appropriate as it's a noun, but the interface 'Area' should be renamed to something more descriptive and noun-like, such as 'AreaCalculable' or 'Shape', to better represent its purpose.